.semaphore/release.yml

```diff
@@ -0,0 +1,29 @@
+version: v1.0
+name: Paprika official release
+agent:
+  machine:
+    type: e1-standard-4
+    os_image: ubuntu1804
+blocks:
+  - name: "🍳 Release"
+    task:
+      secrets:
+        - name: github-ssh
+        - name: paprika-env
+      prologue:
+        commands:
+          - checkout
+          - cache restore client-node-modules-$SEMAPHORE_GIT_BRANCH-$(checksum yarn.lock),client-node-modules-$SEMAPHORE_GIT_BRANCH,client-node-modules-master
+          - chmod 0600 ~/keys/github.key
+          - ssh-add ~/keys/github.key
+          - git config user.name "Paprika Semaphore 2.0"
+          - git config user.email "allison.chen@wegalvanize.com"
+          - echo "//registry.npmjs.org/:_authToken=$NPM_PUBLISH_TOKEN" >> .npmrc
+      jobs:
+        - name: "Publish to NPM"
+          commands:
+            - nvm use
+            - yarn
+            - test -f .changeset/pre.json && yarn changeset pre exit
+            - yarn changeset version
+            - git status --porcelain && git add . && git commit -m"[Bot] Paprika Release" --no-verify && yarn changeset publish && git push origin master --follow-tags --no-verify
```

Comment:
Oh `git status --porcelain` would be great for scripting... but what is it being used for here? I dont think it returns `-1` and stops the script

Reply:
We only want to stop the script when there are no changes. Because if there's no changes `git commit` will return errors

Reply:
Oh and `no changes` means there's nothing to release

Reply:
Jamie is right! I missed something!